pages/project-versions/update/index.js

```diff
@@ -16,20 +16,23 @@ module.exports = settings => {
   );
 
   app.use((req, res, next) => {
-    if (req.project.status === 'active') {
+    const isAmendment = req.project.status !== 'inactive';
+    if (isAmendment) {
       req.breadcrumb('project.amend');
     } else {
       req.breadcrumb('project.update');
     }
     const openTask = get(req.project, 'openTasks[0]');
     const showComments = req.version.status !== 'granted' && !!openTask;
+    const previousVersion = req.project.versions.slice(1).shift();
```

Comment:
Now I know they're ordered the other way it doesn't make sense to do this dance. Just get the second one.

```suggestion
    const previousVersion = req.project.versions[1];
```

Reply:
OOOH...